OrcScala/.settings/org.scala-ide.sdt.core.prefs

```diff
@@ -58,7 +58,7 @@ g=vars
 nameHashing=false
 no-specialization=false
 nowarn=false
-optimise=true
+optimise=false
```

Comment:
I think I did it because several major projects warned that it was buggy, but I'll turn it back on. That was a year ago.